lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -909,6 +909,9 @@ NativeCodeGenerator::CodeGen(PageAllocator * pageAllocator, CodeGenWorkItem* wor
             Js::Throw::OutOfMemory();
         case VBSERR_OutOfStack:
             throw Js::StackOverflowException();
+        case E_ACCESSDENIED:
+            // OOP JIT TODO: if server side can't handle request any more, use better error code and turn off JIT
+            throw Js::JITOperationFailedException(workItem->codeGenResult);
```

Comment:
I'm removing this, for client side just use fatal error for now. we can change to turning off JIT in some cases(like JIT server is dead) in the future. 

---
In reply to: [81824509](https://github.com/Microsoft/ChakraCore/pull/1689#discussion_r81824509) [](ancestors = 81824509)